game/utils/parser.cpp

```diff
@@ -24,12 +20,12 @@ Vec2 Parser::loadVec2(std::string_view src) {
   return Vec2(v[0],v[1]);
   }
 
-Vec3 Parser::loadVec3(const Daedalus::ZString& src) {
+Vec3 Parser::loadVec3(std::string_view src) {
   if(src=="=")
     return Vec3();
 
   float       v[3] = {};
-  const char* str  = src.c_str();
+  const char* str  = src.data();
```

Comment:
Here might be an issue, with string overflow:
later there is `std::strtof`, that assumes zero terminated string as an input

Reply:
I'm now using `<charconv>`'s [from_chars](https://en.cppreference.com/w/cpp/utility/from_chars) for this. Seems to be the best modern solution.